scrapy_zyte_api/utils.py

```diff
@@ -1,6 +1,10 @@
+from importlib.metadata import version
```

Comment:
TIL

Reply:
I'm not sure we can use it though, https://docs.python.org/3/library/importlib.metadata.html says it's 3.8+, while scrapy-zyte-api declares Python 3.7 support.

Reply:
> scrapy-zyte-api declares Python 3.7 support

That has an easy fix :)